modules/agent_files/files/etc/lightdm/xhost.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+xhost +si:localuser:jenkins-agent
+touch /tmp/xhost_`date +"%T"`
```

Comment:
debug code to help with the problem of knowing when the good configuration is loaded into the X server